docker/install/ubuntu_install_onnx.sh

```diff
@@ -20,9 +20,10 @@ set -e
 set -u
 set -o pipefail
 
-# fix to certain version for now
-pip3 install onnx==1.6.0
-pip3 install onnxruntime==1.0.0
+# We need to fix the onnx version because changing versions tends to break tests
+# TODO(mbrookhart): periodically update
+pip3 install onnx==1.8.0
```

Comment:
can you also update `python/gen_requirements.py`?

Reply:
I'm not sure what there is to update? gen_requirements.py doesn't constrain the version of ONNX, and the goal here is to maintain backwards compatibility with old versions of onnx, that's why the importer supports opsets.

Reply:
ah ignore me, we were constraining this separately